### ISSUE TITLE
Tune.handleOpen with Swift 4

### ISSUE THREAD

**Issue by domenicovoto** [TuneOSS/cocoapod2.0#12]
func application(_ app: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey : Any] = [:]) -> Bool {
        Tune.handleOpen(url, options: options)
        return true
    }

Cannot convert value of type '[UIApplicationOpenURLOptionsKey : Any]' to expected argument type '[String : Any]'

Please review the code and also the guide on the website that is not updated.
This method is deprecated starting iOS 9

func application(_ application: UIApplication, open url: URL, sourceApplication: String ? , annotation : Any) - > Bool {
        // when the app is opened due to a deep link, call the Tune deep link setter
        Tune.handleOpen(url, sourceApplication: sourceApplication)
        return true;
    }

**Comment by john-gu** [comment]
Hi @domenicovoto, thank you for bringing this up. We are aware of this issue and the current workaround is:
```
func application(_ app: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey : Any] = [:]) -> Bool {
    // Convert keys from UIApplicationOpenURLOptionsKey -> String.  It's already a String internally
    var newOptions: [String: Any] = [:]
    for (key, value) in options {
        newOptions[String(describing:key)] = value
    }
        
    Tune.handleOpen(url, options:newOptions)
        
    return true
}
```

We have an open ticket to fix this. We will also review the guide on the sample code.

Thanks!

**Comment by ernesttune** [comment]
This is fixed in the 6.0 release.
